### PR TITLE
Fix for #341, added an expose config item on services which defaults to true and is used to set the expose label.

### PR DESCRIPTION
**Pull request by chirino**

### CODE REVIEW

File: core/src/main/java/io/fabric8/maven/core/config/ServiceConfig.java [hunk 1]
```diff
@@ -39,6 +39,10 @@
     @Parameter
     private boolean headless = false;
 
+    // If the expose annotation is added to the service
```
Comment (jimmidyson) [inline core/src/main/java/io/fabric8/maven/core/config/ServiceConfig.java]: expose label, not annotation

File: core/src/main/java/io/fabric8/maven/core/config/ServiceConfig.java [hunk 2]
```diff
@@ -79,6 +87,11 @@ public Builder headless(boolean headless) {
             return this;
         }
 
+        public Builder expose(boolean headless) {
```
Comment (jimmidyson) [inline core/src/main/java/io/fabric8/maven/core/config/ServiceConfig.java]: Paramater name should be `expose`.

File: core/src/main/java/io/fabric8/maven/core/handler/ServiceHandler.java [hunk 3]
```diff
@@ -99,7 +100,7 @@ public Service getService(ServiceConfig service) {
         return ret;
     }
 
-    private Map<String, String> getPrometheusAnnotations(ServiceConfig service) {
+    private Map<String, String> getAnnotations(ServiceConfig service) {
```
Comment (jimmidyson) [inline core/src/main/java/io/fabric8/maven/core/handler/ServiceHandler.java]: Why rename this method?

Comment (chirino) [inline core/src/main/java/io/fabric8/maven/core/handler/ServiceHandler.java]: This method could add other annotation besides prometheus ones.

**Comment by chirino** [comment]
Updated and squashed based on comments.

**Comment by jimmidyson** [comment]
LGTM [merge]

**Pull request merged**